intertidal/composites.py

```diff
@@ -167,7 +215,11 @@ def tidal_composites(
     ds_high_masked = keep_good_only(x=ds_high, where=high_mask.sel(time=high_keep))
 
     # Calculate low and high tide geomedians
-    num_threads = os.cpu_count() - 2
+    if cpus is None:
+        num_threads = os.cpu_count() - 2
+    else:
+        num_threads = cpus - 2
```

Comment:
I don't think we need to subtract two here - if the user is passing in a number of CPUs we should honor that directly
```suggestion
        num_threads = cpus
```